File: clients/src/main/java/com/ynet/belink/common/BelinkException.java
package com.ynet.belink.common;

/**
 * Created by goldratio on 27/06/2017.
 */
public class BelinkException extends RuntimeException {

    private final static long serialVersionUID = 1L;

    public BelinkException(String message, Throwable cause) {
        super(message, cause);
    }

    public BelinkException(String message) {
        super(message);
    }

    public BelinkException(Throwable cause) {
        super(cause);
    }

    public BelinkException() {
        super();
    }

}

File: core/src/main/scala/belink/common/BelinkException.scala
package belink.common

/**
  * Created by goldratio on 30/06/2017.
  */
class BelinkException(message: String, t: Throwable) extends RuntimeException(message, t) {
  def this(message: String) = this(message, null)
  def this(t: Throwable) = this("", t)
}
